docs/sources/next/javascript-api/k6-experimental/fs/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'fs'
+excerpt: 'k6 fs experimental API'
+weight: 10
+---
+
+# fs
+
+{{< docs/shared source="k6" lookup="experimental-module.md" version="<K6_VERSION>" >}}
+
+The k6 filesystem experimental module provides a memory-efficient way to handle file interactions within your test scripts. It currently offers support for opening files, reading their content, and seeking through their content, and retrieving metadata about them.
+
+### Memory efficiency
+
+One of the key advantages of the filesystem module is its memory efficiency. Unlike the traditional [open]() function, which loads a file multiple times into memory, the filesystem module reduces memory usage by loading the file as little possible, and sharing the same memory space between all VUs. This approach signally reduces the memory footprint of your test script, and lets you load and process large files without running out of memory.
+
+### Notes on usage
+
+An important consideration when using the filesystem module is its handling of external file modifications. Once a file is loaded in k6, it behaves like a "view" over the content of the file. This means that if the underlying file is modified externally during a test, those changes won't be reflected in the loaded [File]() instance.
```

Comment:
I guess a link in `[File]()` is missing. Also, it seems like the other files have a similar issue.